libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -23,7 +23,7 @@
             "default": "dialog.result"
         },
         "recognizer": {
-            "$kind": "Microsoft.Recognizer",
+            "$kind": "Microsoft.IRecognizer",
             "title": "Recognizer",
```

Comment:
Should not have been renamed.

Reply:
Looks like you decided to keep the IX in the schema name to correspond to ABC X.

---
In reply to: [395802524](https://github.com/microsoft/botbuilder-dotnet/pull/3602#discussion_r395802524) [](ancestors = 395802524)